docs/source/guide/helm_values.md

```diff
@@ -216,21 +223,24 @@ Parameters specific to the `rqworkers` service of your Label Studio Enterprise d
 | `rqworker.topologySpreadConstraints`                         | list     | [ ]      | Topology Spread Constraints for pod assignment                                              |
-| `rqworker.dnsPolicy`                                         | string   | ""       | Pod DNS policy                                                                              |
+| `rqworker.dnsPolicy`                                         | string   | [ ]      | Pod DNS policy                                                                              |
 | `rqworker.enableServiceLinks`                                | boolean  | false    | Service environment variables                                                               |
 | `rqworker.shareProcessNamespace`                             | boolean  | false    | Enable shared process namespace in a pod                                                    |
-| `rqworker.automountServiceAccountToken`                      | bollean  | true     | Automount service account token for the server service account                              |
+| `rqworker.automountServiceAccountToken`                      | boolean  | true     | Automount service account token for the server service account                              |
+
 
 ## Deprecated parameters
 
 Deprecated parameters no longer in use or supported by Label Studio Enterprise Helm charts. 
 
+<i> Table 4: Deprecated parameters. </i>
+
 | Key                               | Type   | Default Value | Description               |
 |-----------------------------------|--------|---------------|---------------------------|
-| `minio.enabled`                   | string | true          | Enable minio deployment   |
-| `minio.accessKey.password`        | string | minio         | Minio accesskey           |
-| `minio.secretKey.password`        | string | minio123      | Minio accessPassword      |
-| `minio.resources.requests.cpu`    | string | 250m          | Minio requested resources |
-| `minio.resources.requests.memory` | string | 256Mi         | Minio limits resources    |
-| `minio.mode`                      | string | standalone    | Minio mode                |
-| `minio.persistence.enabled`       | string | false         | Minio enable persistence  |
+| `minio.enabled`                   | string | true          | Enable minio deployment.   |
+| `minio.accessKey.password`        | string | minio         | Minio accesskey.           |
+| `minio.secretKey.password`        | string | minio123      | Minio accessPassword.      |
+| `minio.resources.requests.cpu`    | string | 250m          | Minio requested resources. |
+| `minio.resources.requests.memory` | string | 256Mi         | Minio limits resources.    |
+| `minio.mode`                      | string | standalone    | Minio mode.                |
+| `minio.persistence.enabled`       | string | false         | Minio enable persistence.  |
```

Comment:
Unfortunately I can't be sure that the values in all these tables are not damaged, there are too many changes, better to keep this file helm_values.md untouched